### PR TITLE
hook into http ready and start primus

### PR DESCRIPTION
**Pull request by aclave1**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const Promise = require('bluebird');
```
Comment (jaumard) [inline index.js]: you don't need bluebird as we are in ES2016 Promise are native, we don't use bluebird

File: index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const Trailpack = require('trailpack')
+const Primus = require('primus');
+
+const primusDefaults = {
```
Comment (jaumard) [inline index.js:6]: This should be externalize to give the user the possibility to configure primus and choose his web socket lib (not force engine.io)

**Comment by jaumard** [comment]
Can you update this PR to make it work with last Trails version ? Some tests could be nice too if possible

File: package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  "keywords": [
+    "trailpack",
+    "trails",
+    "trailjs"
+  ],
+  "dependencies": {
+    "bluebird": "^3.1.1",
+    "engine.io": "^1.6.7",
+    "primus": "^4.0.4",
+    "trailpack": "latest"
```
Comment (jaumard) [inline package.json]: A little bit dangerous to set it to latest because it can crash anytime we update with an api change

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+language: node_js
+node_js:
+- '4'
+- stable
```
Comment (jaumard) [inline .travis.yml]: Can you change this to have node 4/5 and 6 like other trailpacks please ?

**Comment by jaumard** [comment]
@aclave1 there some problem with TravisCI builds, I think it's because of `eslint`, you need to use the v2 not v1

File: package.json (hunk 3 above)
Comment (aclave1) [inline package.json]: agreed, that's a holdover from the past. I will fix that and remove the bluebird dependency.

File: .travis.yml (hunk 4 above)
Comment (aclave1) [inline .travis.yml]: Yes

**Comment by jaumard** [comment]
Thanks @aclave1 ! :D

**Pull request merged**